features/importing/archivist.feature

```diff
@@ -225,3 +225,36 @@ Feature: Archivist bulk imports
       And I am logged in as "OpenDoors"
     When I go to the Open Doors tools page
     Then I should see "Update Redirect URL"
+
+  Scenario: Open Doors committee members can block an email address from having imports
+    Given I have an Open Doors committee member "OpenDoors"
+      And I have an archivist "archivist"
+      And the default ratings exist
+      And I am logged in as "OpenDoors"
+    When I go to the Open Doors tools page
+    Then I fill in "external_author_email" with "random@example.com"
+    When I submit with the 2nd button
+    Then I should see "We have saved and blocked the email address random@example.com"
+    When I am logged in as "archivist"
+      And I go to the import page
```

Comment:
I think you can replace lines 239-244 with just `And I import the work "http://ao3testing.dreamwidth.org/593.html" by "ao3testing" with email "random@example.com"` here. It does check the "Post without previewing" option, though, so if for some reason that won't work, you could do this instead:
```
And I start to import the work "http://ao3testing.dreamwidth.org/593.html" by "ao3testing" with email "random@example.com"
And I press "Import"
```